docs/deploy.rst

```diff
@@ -63,6 +63,9 @@ Nginx can be used by adding a block such as:
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
         proxy_pass http://localhost:9000/;
+        # Must turn off response buffering for SSE to work
+        proxy_buffering off;
+        proxy_read_timeout 600s;
```

Comment:
We shouldn't need `proxy_connect_timeout` since the documentation says that's only for how long it takes to make the initial connection. I don't see `proxy_write_timeout` in the documentation (I am looking at http://nginx.org/en/docs/http/ngx_http_proxy_module.html), is that actually an option?